entrypoint.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh -l
 
+if [ -n "$5" -a -n "$6" -a -n "$7" -a -n "$8" ]; then
+    helm repo add "$5" "$6" --username "$7" --password "$8"
```

Comment:
Does this really need to be in the shared action? The caller is passing in almost the whole line anyway